### PR TITLE
doc: documentation to override stylesheet

### PR DESCRIPTION
**Pull request by mGGk-fr**
# What

This MR implement documentation for style customisation of a theme

### CODE REVIEW

File: docs/02-guides/override-a-stylesheet.mdx
Comment (Kagaminara) [inline docs/02-guides/override-a-stylesheet.mdx:1]: > description:
  This guide explains how to override a component in Front-Commerce, to
  customize the default styles

**issue:** this is not about overriding a component

**suggestion:** update the description

Comment (mGGk-fr) [reply docs/02-guides/override-a-stylesheet.mdx:1]: It has been ported from V2 doc, we must either change the description of the issue to match the content of the doc, or, rework the documentation to only focus on stylesheet overriding and transpose the other knowledge to other guides.

File: docs/02-guides/override-a-stylesheet.mdx [hunk 1]
```diff
@@ -8,9 +8,205 @@ sidebar_position: 2
...
+<Figure caption="Here is an example with the awesome Smashing Magazine website">
+
+![CSS Stats lists all the 62 colors and 59 unique background colors of Smashing Magazine](./assets/smashingmagazine-cssstats.png)
+
+</Figure>
+
+:::info
+
+If you want to learn more about it, you can have a look at
+[Design tokens for dummies](https://uxdesign.cc/design-tokens-for-dummies-8acebf010d71)
```
Comment (Kagaminara) [inline docs/02-guides/override-a-stylesheet.mdx:38]: **note:** in the linked article, an update suggest that the updated version of the article is at this link instead:
https://specifyapp.com/blog/introduction-to-design-tokens

Comment (mGGk-fr) [reply docs/02-guides/override-a-stylesheet.mdx]: updated !

**Review by paul-vd — APPROVED** [review]

**Review by Kagaminara — APPROVED** [review]

File: docs/02-guides/override-a-stylesheet.mdx
Comment (mGGk-fr) [reply docs/02-guides/override-a-stylesheet.mdx:1]: renamed section and description

**Pull request merged**